cmd/crio/main.go

```diff
@@ -412,28 +412,28 @@ func main() {
 		},
 		cli.StringSliceFlag{
 			Name:  "hooks-dir",
-			Usage: "set the OCI hooks directory path (may be set multiple times)",
+			Usage: fmt.Sprintf("set the OCI hooks directory path (may be set multiple times) (default: %q)", defConf.HooksDir),
 		},
 		cli.StringSliceFlag{
 			Name:  "default-mounts",
-			Usage: "add one or more default mount paths in the form host:container (deprecated)",
+			Usage: fmt.Sprintf("add one or more default mount paths in the form host:container (deprecated) (default: %q)", defConf.DefaultMounts),
 		},
 		cli.StringFlag{
 			Name:   "default-mounts-file",
-			Usage:  "path to default mounts file",
+			Usage:  fmt.Sprintf("path to default mounts file (default: %q)", defConf.DefaultMountsFile),
 			Hidden: true,
 		},
 		cli.StringFlag{
 			Name:  "default-capabilities",
-			Usage: "capabilities to add to the containers",
+			Usage: fmt.Sprintf("capabilities to add to the containers (default: %q)", defConf.DefaultCapabilities),
 		},
-		cli.StringFlag{
+		cli.StringSliceFlag{
```

Comment:
I think this clashes with the config merge part here:

https://github.com/cri-o/cri-o/blob/296d80b6ac61b427583e1cfe06e74da73e7e875b/cmd/crio/main.go#L157-L159

It also changes the command line interface (string slices can be specified multiple times, before it was comma separated).

Reply:
Yeah, i fixed that in the latest commit